packages/react-aria-components/src/Disclosure.tsx

```diff
@@ -181,7 +181,15 @@ function Disclosure(props: DisclosureProps, ref: ForwardedRef<HTMLDivElement>) {
   );
 }
 
-export interface DisclosurePanelProps extends RenderProps<{}> {
+export interface DisclosurePanelRenderProps {
+  /**
+   * Whether keyboard focus is within the disclosure panel.
+   * @selector [data-focus-visible-within]
+   */
+  isFocusVisibleWithin: boolean
```

Comment:
Should we have anything else in here? Maybe it would be redundant to include isExpanded, etc. since they're already in `DisclosureRenderProps`

Reply:
I think this is good for now since as you mentioned, the Disclosure's own render props can be used to style the panel itself

Reply:
i think this is fine for now. we could always open it up to more things in the future if we get any feedback for it